app/javascript/dashboard/components/ChatList.vue

```diff
@@ -161,7 +168,14 @@ export default {
         (a, b) =>
           this.lastMessage(b).created_at - this.lastMessage(a).created_at
       );
-      return sorted;
+      return sorted.map((chat, index) => {
```

Comment:
Move this to a variable like `sortedWithNextChat` or something ?

Reply:
06a71d3

Reply:
@nithindavid thanks for the review. 😄 

I did all your suggestions, could you please take another look?